vignettes/miaViz.Rmd

```diff
@@ -328,7 +329,7 @@ plotDMNFit(dmn_se, type = "laplace")
 
 # Serial data ordination and trajectories
 
-```{r, error=FALSE, warning=FALSE, results='hide'}
+```{r, error=FALSE, warning=FALSE, results='hide', eval=FALSE}
```

Comment:
Build currently fails "miaTime not found" I guess this is good to have for now (we can also have it behind require)